### PR TITLE
Allow more than one flyleaf with freeform input

### PR DESCRIPTION
**Pull request by acestronautical**
Being able to insert an adjustable number of
flyleafs is important for generating books with
even signature counts

### CODE REVIEW

**Comment by acestronautical** [comment]
Addresses feature: https://github.com/momijizukamori/bookbinder-js/issues/76

File: index.html [hunk 1]
```diff
@@ -380,4 +380,4 @@ <h2>Signature Info</h2>
     </div>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (acestronautical) [inline index.html:383]: Sorry my editor added a newline at the end of this file. It's technically correct to end the file with a newline but I can remove if you really want min-diff.

**Comment by acestronautical** [comment]
image

**Review by sithel — APPROVED** [review]
hate to say it, but I'm struggling to figure out how to check out your branch for testing. fucking forks....  
brain is fried from week of work but sure-- this looks minimal and harmless enough... :shipit:

**Pull request merged**